apps/directory/src/components/Footer.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="container">
+    <div class="col-12>">
+      <footer class="bottom-0 p-0 w-100 text-primary-text">
+        <hr class="mt-10 mb-10" />
+        <div class="row d-flex flex-column flex-md-row">
+          <div class="col text-center mb-4 mb-md-0">
+            <a href="https://bbmri-eric.eu">
+              <img
+                width="auto"
+                height="40"
+                src="/img/bbmri-eric-logo.png"
+                alt="logo negotiator"
+              />
+            </a>
+          </div>
+          <div class="col text-center">
+            <span> Follow Us: </span>
+            <a
+              v-if="isFooterFollowUsVisible"
```

Comment:
I see a lot of the same v-if here, might be nice to add a `<div>` around those and put the if on that.

Reply:
Adding arbitrary <div> tags within the html would break the styling.